src/common/middleware/call-api.js

```diff
@@ -109,20 +109,21 @@ function validateDefaults({ endpoint, csrfToken }) {
 }
 
 function validateSettings(settings) {
-  const { path, options, types } = settings;
+  const { path, types } = settings;
   if (typeof path !== 'string') {
     throw new Error('Specify a string path.');
   }
-  if (!options) {
-    throw new Error('Specify settings for API request');
-  }
```

Comment:
Some of our API calls don't supply any options. It doesn't make sense to enforce them. They're not required by the fetch API, and they shouldn't be required here either.